cli/azd/pkg/infra/provisioning/terraform/terraform_provider.go

```diff
@@ -295,23 +256,15 @@ func (t *TerraformProvider) State(ctx context.Context, options *StateOptions) (*
 }
 
 // Creates the terraform plan CLI arguments
-func (t *TerraformProvider) createPlanArgs(isRemoteBackendConfig bool) []string {
+func (t *TerraformProvider) createPlanArgs() []string {
 	args := []string{fmt.Sprintf("-var-file=%s", t.parametersFilePath())}
 
-	if !isRemoteBackendConfig {
-		args = append(args, fmt.Sprintf("-state=%s", t.localStateFilePath()))
-	}
-
 	return args
 }
 
 // Creates the terraform apply CLI arguments
-func (t *TerraformProvider) createApplyArgs(
-	isRemoteBackendConfig bool, data terraformDeploymentDetails) ([]string, error) {
+func (t *TerraformProvider) createApplyArgs(data terraformDeploymentDetails) ([]string, error) {
 	args := []string{}
-	if !isRemoteBackendConfig {
-		args = append(args, fmt.Sprintf("-state=%s", data.localStateFilePath))
-	}
```

Comment:
How does remote state continue to work while removing the `-state` param from being passed as a TF arg?